### DISCUSSION TITLE
Chips card in new sections

### DISCUSSION THREAD

**Discussion by gwww** [piitaya/lovelace-mushroom#1443] [General]
When I create a new section on a dashboard and put a chips card in the section there is a large white space below the chips card before the title of the next section. 

Details…. Section with chips card is the first section. Only the chips card is in the section. The section with the chips card has no title. 

can the white space be reduced?

![image](https://github.com/piitaya/lovelace-mushroom/assets/3060128/6044531b-c0de-4b88-bce7-644ee88c94ab)

**Comment by BBE-FR** [comment]
From my tests to reproduce the issue, the big space is not "below" the chip card, but "above" the header of the section:

hereafter some screen-shot:
section 1: header+chip 
section 2: chip only
section 3 and 4: header + various cards

When the display is:
section 1 / section 2
section 3 / section 4
![image](https://github.com/user-attachments/assets/7b5ba774-9cda-4652-8325-d7fb58cdc390)

When you reduce screen width to have one columns 
no space below chip in section 1, the space is "above" section 3 header:
![image](https://github.com/user-attachments/assets/33b54fc6-a1cc-4e4b-9179-e60f50ff1f52)
